server/server.lua

```diff
@@ -1,12 +1,31 @@
 QBCore = exports['qb-core']:GetCoreObject()
 -- PSCore = exports['ps-core']:GetCoreObject()
 
+if Config.Inventory == "ox" then
+local function registerOxInventoryStashes()
```

Comment:
nit: not sure if this is the github preview being weird, but can you make sure everything inside this if block is tabbed in so its inside the block